apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -50,6 +50,51 @@
 {{#inline}}
 </span>
 {{/inline}}
+{{/radio}}
+{{^radio}}
+<!-- Weirdly enough putting this in a div causes it to display without the inline -->
+<span class="py-1 d-inline-flex align-items-center">
+    <script>
+        $(() => {
+            $('#pl-multiple-choice-select-{{uuid}}').selectpicker({})
+                .on('rendered.bs.select loaded.bs.select shown.bs.select', (e) => {
+                    // If any of the options has math, render the math as needed
+                    // Events: loaded (when the page opens), rendered (when the value changes),
+                    // shown (when the dropdown extends)
+                    MathJax.startup.promise.then(async () => MathJax.typesetPromise($(e.target).parent()));
+                });
+        });
+    </script>
+    <select name="{{name}}" data-style="btn-outline-secondary"
+            id="pl-multiple-choice-select-{{uuid}}" {{^editable}}disabled{{/editable}}>
+        <option disabled selected value></option>
+        <!-- TODO disable blank option to use if blank is expected -->
```

Comment:
I think I had some issues with allow blank earlier, which is why I put this here. As long as the dropdown behaves as expected during testing I think this can be removed.

Reply:
Looks like this works as-expected, will remove the comment.